.github/workflows/dependabot-update-external-deps.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow is triggered when a pull request is opened by Dependabot
+# This workflow will use metadata from Dependabot update to update external_dependencies.txt and update the dependency across all the SDKs
+# This workflow will auto-approve and merge the pull request if the update is a patch version or minor version update with a compatibility score of 80 or higher.
+
+# NOTE: The auto-approved and merge step is currently disabled.
+name: Dependabot Update External Dependencies
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+  issues: write
+  repository-projects: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+    - name: Fetch Dependabot metadata
+      id: dependabot-metadata
+      uses: dependabot/fetch-metadata@v2
+      with:
+        github-token: "${{ secrets.GITHUB_TOKEN }}"
+        alert-lookup: true
+        compat-lookup: true
+        skip-verification: true
+        skip-commit-verification: true
+
+    - name: Checkout repository
```

Comment:
We should avoid checking out the head ref of a pull request in an action that has write permission. That can potentially cause a security issue.